Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -1227,9 +1227,6 @@ WarpX::ApplyInverseVolumeScalingToCurrentDensity (MultiFab* Jx, MultiFab* Jy, Mu
     const Real dr = dx[0];
```

Comment:
Could you open a separate pull request for the changes contained in this file - since this can be reviewed independently from the rest of the PR?

Feel free to simultaneously keep these changes in the current PR, though, since my understanding is that this is necessary for the CI tests to pass.